workflow-bot-app/src/main/java/com/symphony/bdk/workflow/expiration/WorkflowExpirationInterface.java

```diff
@@ -0,0 +1,11 @@
+package com.symphony.bdk.workflow.expiration;
+
+import com.symphony.bdk.workflow.versioning.model.WorkflowExpirationJob;
+
+import java.time.Instant;
+
+public interface WorkflowExpirationInterface {
```

Comment:
Use Interface as part of the name is not very ideal IMO,  could you change to WorkflowExpirationService, and change the current WorkflowExpirationService implementation to DefaultWorkflowExpirationService.